teamfoundationplugin.cpp

```diff
@@ -219,20 +256,15 @@ bool TeamFoundationPlugin::submitEditorAboutToClose()
 
 void TeamFoundationPlugin::updateActions(VcsBase::VcsBasePlugin::ActionState as)
 {
-    if (!enableMenuAction(as, m_menuAction))
-    {
-        m_commandLocator->setEnabled(false);
-        return;
-    }
+//    m_commandLocator->setEnabled(true);

```

Comment:
m_commandLocator is only used here, I assume you can remove the variable entirely if you remove the use here.